docs/source/index.rst

```diff
@@ -29,6 +19,92 @@ Overall, SciKeras aims at being as flexible as Keras while having a clean interf
    advanced
    api
 
+Differences with ``tf.keras.wrappers.scikit_learn``
+---------------------------------------------------
+
+* Full compatibility with the Scikit-Learn API, including grid searches, ensembles, transformers, etc.
+* Support for Keras Functional and Subclassed Models.
+* Support for pre-trained models.
+* Support for dynamically set Keras parameters depending on inputs (e.g. input shape).
+* Support for hyperparameter tuning of optimizers and losses.
+* Support for multi-input and multi-ouput Keras models.
+* Functional `random_state` for reproducible training.
+* Many more that you will discover as you use SciKeras!
+
+
+Transitioning from ``tf.keras.wrappers.scikit_learn``
+-----------------------------------------------------
+
+SciKeras is largely backwards compatible with the existing wrappers. For most cases, you can just change your import statement from:
+
+.. code:: diff
+
+   - from tensorflow.keras.wrappers.scikit_learn import KerasClassifier, KerasRegressor
+   + from scikeras.wrappers import KerasClassifier, KerasRegressor
+
+
+SciKeras does however have some backward incompatible changes:
+
+One-hot encoding of targets for categorical crossentropy losses
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SciKeras will not longer implicitly inspect your Model's loss function to determine if
+it needs to one-hot encode your target to match the loss function. Instead, you must explicitly
+pass your loss function to the constructor:
+
+.. code:: python
+
+   clf = KerasClassifier(loss="categorical_crossentropy")  # or loss=CategoricalCrossentropy(), etc.
+
+Removal of ``**kwargs`` from fit and predict
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a future release of SciKeras, ``**kwargs`` will be removed from fit and predict. To future
+proof your code, you should instead declare these parameters in your constructor:
+
+.. code:: diff
+
+   - clf = KerasClassifier(...)
+   + clf = KerasClassifier(..., batch_size=32)
+   - clf.fit(..., batch_size=32)
+   + clf.fit(...)
+
+Or to declare separate values for ``fit`` and ``predict``:
+
+.. code:: python
+
+   clf = KerasClassifier(fit__batch_size=32, predict__batch_size=10000)
+
+Renaming of ``build_fn`` to ``model``
```

Comment:
Are these warnings in `index.rst` necessary? I'm having a hard time seeing the reason to feature the following on the front page of the documentation:

* Removal of keyword args
* Renaming `build_fn` to `model`
* Model introspection

All of those issue warnings with tips on how to resolve. Why are they on the front page of the documentation? When I write docs, the front page always describes why a user should use my software, not issues they might face.

I might dedicate a separate page to potential hotspots when transitioning from TF to SciKeras.

Reply:
Yeah a separate page should work and sounds good. Thanks.